.github/pull_request_template.md

```diff
@@ -20,11 +20,6 @@ If this PR closes an issue, add '<AB#213700>' somewhere in the PR summary. As a
 - [ ] This PR contains tests
 - [ ] Version number in [package.json](https://github.com/DEFRA/adp-portal/blob/main/app/package.json#L3) has been updated
 
-# Dependency check (tick or delete)
```

Comment:
this should go back in?

Reply:
That must've gotten deleted whilst I was reverting back to my old changes, I'll add that back in now, thank you :)